bundles/io/org.eclipse.smarthome.io.rest.core/src/main/java/org/eclipse/smarthome/io/rest/core/discovery/DiscoveryResource.java

```diff
@@ -91,7 +101,9 @@ public void onFinished() {
             }
         });
 
-        return Response.ok(discoveryServiceRegistry.getMaxScanTimeout(bindingId)).build();
+        DiscoveryServiceDTO dto = new DiscoveryServiceDTO();
+        dto.timeout = discoveryServiceRegistry.getMaxScanTimeout(bindingId);
```

Comment:
I don’t think it’s too much hassle - just to know what to call the class - other than that it’s no work I think. Even if we can’t see a use for additional return information, IMHO it would be wise to generally use a format that allows future enhancements rather than making it impossible. I guess when the discovery interface was written, no-one thought of returning the timeout, so I don’t think we can say we will never want to return other information.